pyproject.toml

```diff
@@ -70,6 +70,7 @@ dependencies = [
     "torchvision>=0.13",
     "tqdm",
     "xmltodict",
+    "myst-parser",
```

Comment:
I think this belongs below in `project.optional-dependencies` with the rest of the docs dependencies